### ISSUE TITLE
enum's not generated for base_locale_empty_string 

### ISSUE THREAD

**Issue by yagmure15** [slang-i18n/slang#182]
**Describe the bug**
I would like to sincerely thank you for the development of base_locale_empty_string. Here, I have made the necessary additions in the English JSON file. I'm also adding other languages as 'empty' just like in the examples. This way, the person doing the translations understands that they need to translate it. When I initially send the section parts like in the first example, there's no problem. However, when the translator translates some parts and not others in the second example, it becomes an issue. 

*Example 1*
**strings_en.i18n.json**
```
    "sections(context=Sections, param=sections)": {
      "deviceInfo": "Device Info",
      "identification": "Identification",
      "information": "Information",
      "notes": "Notes",
      "purchaseInfo": "Purchase Info"
    },
```

**strings_ro.i18n.json**
```
    "sections(context=Sections, param=sections)": {
      "deviceInfo": "",
      "identification": "",
      "information": "",
      "notes": "",
      "purchaseInfo": ""
    },
```

*Example 2*
**strings_en.i18n.json**
```
    "sections(context=Sections, param=sections)": {
      "deviceInfo": "Device Info",
      "identification": "Identification",
      "information": "Information",
      "notes": "Notes",
      "purchaseInfo": "Purchase Info"
    },
```

**strings_ro.i18n.json**
```
    "sections(context=Sections, param=sections)": {
      "deviceInfo": "Informații dispozitiv",
      "identification": "Identificare",
      "information": "",
      "notes": "notes",
      "purchaseInfo": "Informații achiziție"
    },
```

**Error Message**
Failed to build iOS app
Error (Xcode): lib/translations/strings.g.dart:3819:11: Error: The type 'Sections' is not exhaustively matched by the switch cases since it doesn't match 'Sections.information'.

**Comment by Tienisto** [comment]
You will need to remove the whole context node. Currently, each language is generated separately from each other, so "ro" does not know how many strings in "en" exist

**Comment by yagmure15** [comment]
The context node is very useful for us; the slang library is one of the reasons we prefer it. Otherwise, the code becomes more complicated. The likelihood of encountering the situation in the second example is very high, and in that case, the application crashes. At the very least, to prevent the application from crashing, **a default "" (empty string)** can be provided in the _string.g.dart_ file.


**_string.g.dart file_**
```
.
.
.
  @override
  String sections({required Sections sections}) {
    switch (sections) {
      case Sections.deviceInfo:
        return 'Informații dispozitiv';
      case Sections.identification:
        return 'Identificare';
      case Sections.notes:
        return 'Note';
      case Sections.purchaseInfo:
        return 'Informații achiziție';
      default:
         return '';
    }
  }
.
.
.
```

Even in this case, I will have to find the empty value in the generated files and process it. I am considering replacing **"" (the empty values)** with an expression like **"untranslated key -> $keyvalue."** That's the general situation. What do you think? Is there another way?

**Comment by talhakerpicci** [comment]
Im having the same issue. Any plans on handling this case?

**Comment by Tienisto** [comment]
Fixed in `v3.29.0`

**Comment by talhakerpicci** [comment]
When we tried to upgrade the package, we encountered this issue: [issue](https://github.com/slang-i18n/slang/issues/247)

What changes have been made and what/how should we implement it? I would be happy if you can navigate me to find out whats the problem.
